.circleci/config.yml

```diff
@@ -193,8 +193,15 @@ jobs:
       - run:
           name: 'Unit Tests'
           command: node build-system/pr-check/unit-tests.js
+          environment:
+            JUNIT_REPORT_PATH: result-reports/
+            JUNIT_REPORT_NAME: unit.xml
       - fail_fast
       - teardown_vm
+      - store_artifacts:
+          path: result-reports
+      - store_test_results:
+          path: result-reports
```

Comment:
Can we capture these two as a new command in the `commands` section above? Then it will become easier to reuse for other testing jobs like E2E, Experiment A/B/C, Module / Nomodule (integration), etc.

Reply:
Yes, great suggestion